lib/rspec/mocks/proxy.rb

```diff
@@ -228,6 +228,11 @@ def prepended_modules_of_singleton_class
         end
       end
 
+      # @private
+      def method_double_if_exists_for_message(message)
```

Comment:
Putting `if` in the method name kinda rubs me the wrong way.  WDYT of `existing_method_double_for(message)` instead?